docker-test/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM jimmycuadra/rust:1.15.0
+
+RUN apt-get update && apt-get install -y \
+    libdbus-1-3 \
+    libdbus-1-dev \
+    dbus-x11 \
+    python3 \
+    python3-pip \
+    libssl-dev \
+    openssl \
+   && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 install canonicaljson
+
+ARG bin_dir=/usr/local/bin
+ARG conf_dir=/usr/local/etc
+
+COPY canonical_json.py $bin_dir
```

Comment:
Nop. The base image I'm using requires '/usr/bin/python3' instead of '/usr/bin/python' as the interpreter path in canonical_json.py. I'm hoping anyway that we blow this whole thing away in PRO-2742, so it's a temporary workaround.